packages/xrpl/HISTORY.md

```diff
@@ -1,8 +1,9 @@
 # xrpl.js (ripple-lib) Release History
 
 Subscribe to [the **xrpl-announce** mailing list](https://groups.google.com/g/xrpl-announce) for release announcements. We recommend that xrpl.js (ripple-lib) users stay up-to-date with the latest stable release.
-
 ## Unreleased
+
```

Comment:
```suggestion
## Unreleased
- (No unreleased changes yet; future PRs should update this line)

```

Reply:
I think this will be another thing that's easy to forget to add/update. I'd rather just leave it empty to avoid accidentally including this in the HISTORY because it went unnoticed.